app/views/account/posts/_gallery.html.slim

```diff
@@ -0,0 +1,36 @@
+.gallery.container
+  span.close[type="button" data-dismiss="modal" aria-label="Close"]
+    i.fa.fa-window-close
+  .row
+    .col-lg-8
+      .post-photo
+        .post-actions
+          .previous-post
+            = link_to '<<', account_post_path(post.next), method: :get, remote: true unless post.next == nil
```

Comment:
please use method `nil?`

Reply:
i think it because of JS background :), i will try to use rails methods for similar actions